code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -1,3 +1,5 @@
+#define STABILIZED_LIGHT_PINK_TRAIT "stabilized_light_pink"
```

Comment:
Could ya cram this somewhere in code/__DEFINES/traits.dm pretty please? It keeps all the trait sources in one tidy place for quicker/easier reference and allows future coders to reference this source from outside of the code in the future.